### DISCUSSION TITLE
第3节课笔记(2班)

### DISCUSSION THREAD

**Discussion by MING-ZCH** [InternLM/Tutorial#181] [笔记]
## 2班第三节课笔记交这里哟~
* 链接形式提交
* 也记得在信息收集文档对应作业处填写链接

**Comment by jinxianwei** [comment]
[第三节课笔记](https://blog.csdn.net/jinxianwei1999/article/details/135462990?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135462990%22%2C%22source%22%3A%22jinxianwei1999%22%7D)

**Comment by yangff0016** [comment]
[第3节课笔记](https://blog.csdn.net/weixin_42837310/article/details/135466106?spm=1001.2014.3001.5502)

**Comment by songyuc** [comment]
[《基于 InternLM 和 LangChain 搭建你的知识库》学习笔记](https://blog.csdn.net/songyuc/article/details/135466277)

**Comment by linmiaoyan** [comment]
[第三节课笔记](https://www.bilibili.com/read/cv29201869/?spm_id_from=333.999.0.0&jump_opus=1)

**Comment by DarthThomas** [comment]
[第三课笔记](https://share-tianhe.notion.site/93377c8775884a32aac73bdf8b721f08?pvs=4)

**Comment by pranerd** [comment]
[第三节课笔记](https://blog.csdn.net/gaxzj/article/details/135462859?spm=1001.2014.3001.5501)

**Comment by xulinui** [comment]
https://h90ag9106t.feishu.cn/docx/OeyFdw1ZYosnC3xVia5csGkpnsg

**Comment by qihong2019** [comment]
[第3节 基于 InternLM 和LangChain 搭建你的知识库](https://www.processon.com/view/link/65a14558320d8b2cc6714aa2)

![第3节 基于 InternLM 和LangChain 搭建你的知识库](https://github.com/InternLM/tutorial/assets/47970679/ee07cf94-8689-4aa1-ba6b-944283020625)

**Comment by ljqiff** [comment]
https://blog.csdn.net/qq_25820969/article/details/135563912?spm=1001.2014.3001.5502

**Comment by kyrie2to11** [comment]
[InterLM L-3: 基于 InternLM 和 LangChain 搭建知识库](https://zhuanlan.zhihu.com/p/676897517) 打卡~

**Comment by kishiokon** [comment]
[基于 InternLM 和 LangChain 搭建你的知识库](https://www.bilibili.com/read/cv29496122/?jump_opus=1)

**Comment by CoderJackZhu** [comment]
[第三节笔记](https://zhuanlan.zhihu.com/p/677658292)

**Comment by phdbrianlee** [comment]
[InternLM X LangChain 4 知识库RAG](https://www.jianshu.com/p/fd93a79000af?v=1705215549921)

**Comment by WanpengXu** [comment]
[书生·浦语大模型实战营-第三节](https://b07ofnm9xj7.feishu.cn/wiki/YuJ8whwO2irzgskoRkfcxBwdnHb)

**Comment by Mr-Vincenyang** [comment]
[第三节课笔记](https://zhuanlan.zhihu.com/p/677682069)

**Comment by YanxingLiu** [comment]
第三节课笔记 https://s5eixc6gba.feishu.cn/docx/XR7Uda5VGoDpKmx5fYuc6RS6nOc?from=from_copylink

**Comment by zyy0831** [comment]
第三节课笔记 https://www.yuque.com/yuanyang-xb8hd/eeag2y/pix39e42n4c4gc1g

**Comment by KylinCarol** [comment]
第三课笔记：https://blog.csdn.net/m0_65412000/article/details/135747096

**Comment by PrinceRay7** [comment]
第三节课笔记
https://blog.csdn.net/Prince_Ray/article/details/135832127?spm=1001.2014.3001.5501
